server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserMailboxesRoutesTest.java

```diff
@@ -149,16 +156,16 @@ void getMailboxesShouldUserErrorFoundWithNonExistingUser() throws Exception {
             Map<String, Object> errors = when()
                 .get()
             .then()
-                .statusCode(HttpStatus.NOT_FOUND_404)
-                .contentType(ContentType.JSON)
+                .statusCode(NOT_FOUND_404)
+                .contentType(JSON)
                 .extract()
                 .body()
                 .jsonPath()
                 .getMap(".");
 
             assertThat(errors)
-                .containsEntry("statusCode", HttpStatus.NOT_FOUND_404)
-                .containsEntry("type", "InvalidArgument")
+                .containsEntry("statusCode", NOT_FOUND_404)
+                .containsEntry("type", ErrorType.NOT_FOUND.getType())
```

Comment:
We should not reference constant in tests, I prefer `"notFound"`

Reply:
well, i will replace it by own constant at this class, are you ok with it?